src/components/navbar/Wallet.tsx

```diff
@@ -87,44 +80,31 @@ const AccountContainer = ({ account }: { account?: string }) => {
       }}
     >
       <div className={walletAccount}>
-        <AccentText extraSmall={xs} color="text-primary-semi-dimmed">
-          {needNetworkChange
-            ? 'Change network'
-            : xs
-            ? 'No wallet'
-            : 'No wallet connected'}
-        </AccentText>
+        <UnconnectedAddress needNetworkChange={needNetworkChange} />
       </div>
       <div className={width('w-fit')}>
-        {xs ? (
-          <SmallSeal connected={false} />
-        ) : (
-          <SealWallet connected={false} />
-        )}
+        <UnconnectedLogo />
       </div>
     </div>
   )
 }
 
 export default function () {
   const { account } = useSnapshot(WalletStore)
-  const { md } = useBreakpoints()
 
   return (
     <div className={walletContainer}>
-      {md && (
-        <>
-          <div className={socialContainer}>
-            <SocialLink url="https://discord.gg/NHk96pPZUV">
-              <Discord />
-            </SocialLink>
-            <SocialLink url="https://twitter.com/bigwhalelabs">
-              <Twitter />
-            </SocialLink>
-          </div>
+      <span className={displayFromMd}>
+        <div className={socialContainer}>
```

Comment:
Hm, maybe we should include `displayFromMd` into `socialContainer` and into `delimiterContainer`, less nesting is better, I think

Reply:
Mixed them